drivers/rudex.js

```diff
@@ -0,0 +1,22 @@
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { parseToFloat } = require('../lib/utils.js');
+
+module.exports = async () => {
+  const markets = await request('https://ticker.rudex.org/api/v1/ticker');
+
+  return Object.keys(markets).map((market) => {
+    const [quote, base] = market.split('_'); // reversed
```

Comment:
Mh I don't think they are reversed. If you look at this ticker for example, the `last_price` is really the `BTC` price in `USD`:
```
"BTC_USDT": {
  "base_id": 825,
  "quote_id": 1,
  "last_price": "5586.84030939",
  "quote_volume": "0.71047096",
  "base_volume": "3752.79278600",
  "isfrozen": 0,
  "lowest_ask": "5575.12113344",
  "highest_bid": "5639.54244136",
  "percent_change": "9.37",
  "updated_at": "2020-03-19T11:11:45.000Z"
}
```